pkgs/os-specific/linux/libzbd/default.nix

```
@@ -0,0 +1,40 @@
+{
+  autoconf-archive,
+  autoreconfHook,
+  fetchFromGitHub,
+  gtk3,
+  lib,
+  libtool,
+  pkg-config,
+  stdenv,
+  guiSupport ? false,
+}:
+stdenv.mkDerivation rec {
+  pname = "libzbd";
+  version = "2.0.3";
+
+  src = fetchFromGitHub {
+    owner = "westerndigitalcorporation";
+    repo = "libzbd";
+    rev = "v${version}";
+    sha256 = "GoCHwuR4ylyaN/FskIqKyAPe2A2O3iFVcI3UxPlqvtk=";
+  };
+
+  nativeBuildInputs = [
+    autoconf-archive # this can be removed with the next release
+    autoreconfHook
+    libtool
+  ];
+
+  buildInputs = lib.optionals guiSupport [gtk3 pkg-config];
```

```suggestion
  buildInputs = lib.optionals guiSupport [ gtk3 ];
```
pkg-config goes to nativeBuildInputs.